commons/phone/phone_contacts_list.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactsList",
+  "doc": "Number of new contacts added/removed from the list of contacts between this and the previous record.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "device timestamp in UTC (s)"},
+    {"name": "timeReceived", "type": "double", "doc": "device receiver timestamp in UTC (s)"},
+    {"name": "contactsAdded", "type": "int", "doc": "Number of new contacts added to the list of contacts."},
```

Comment:
perhaps make the type `["null", "int"]`: the first time you scan this number is unknown? To the doc, add something like

> Number of new contacts added to the list of contacts _since the previous measurement that was sent. The value is null if this is the first measurement or if the previous measurement is unknown to the sender._

Same for `contactsRemoved`.

Reply:
Done